.appveyor.yml

```diff
@@ -19,20 +24,44 @@ configuration:
   #only:
     #- master
 
-before_build:
-  - mkdir build
-  - cd build
+install:
   - set PATH=%QTDIR%\bin;%PATH%
+  - set Qt5_DIR=%QTDIR%\lib\cmake\Qt5
   - set PATH=C:\MinGW\bin;C:\MinGW\msys\1.0;%PATH%
   - set PATH=%PATH:C:\Program Files\Git\usr\bin=% # trick to remove sh.exe
-  - cmake "-G%GENERATOR%" -DCMAKE_PREFIX_PATH="%QTDIR/lib/cmake/Qt5" ..
+  - mkdir dependencies
+  - cd dependencies
+  - mkdir install
+  - set CMAKE_INSTALL_PREFIX=%cd%\install
+  - set DEPENDENCIES=%cd%\install
+  - git clone https://github.com/catchorg/Catch2.git
+  - cd Catch2
+  - mkdir build
+  - cd build
+  - cmake "-G%GENERATOR%" -DBUILD_TESTING=OFF ..
+  - cmake --build . --target install
+  - set Catch2_DIR=%DEPENDENCIES=%cd%\install\lib\cmake\Catch2
+  - cd ..
+  - cd ..
+  - cd ..
```

Comment:
It might have been much easier to pull the single-header catch into the sources.

Reply:
True, it probably would have been easier. I did it this way because the canonical way to declare a dependency in modern CMake is with `find_package`.